src/status_im/ui/screens/db.cljs

```diff
@@ -33,6 +33,7 @@
              :tags                       []
              :sync-state                 :done
              :wallet                     {}
+             :wallet.transactions        constants/default-wallet-transactions
```

Comment:
why to not add in wallet map? `:wallet   {:transactions        constants/default-wallet-transactions}`

Reply:
That's an option. Is the latter better, or our convention?

Reply:
i think @yenda has something to say here :)

Reply:
I think it is better to have the wallet related stuff in the wallet map yes

Reply:
Not straightforward now as we cleanup the whole `wallet` key while initializing app-db.
Not quite sure what would be the proper way to address that so will skip for now.

Reply:
Is the reason we are cleaning up the whole wallet key not because you might switch account? Wouldn't that imply that we want transactions cleared as well then? Or am I misunderstanding?

Reply:
Yes why is it a problem ? If there is default filters in default app-db there is nothing to do

Reply:
@oskarth What I want to store in `:transactions` is mostly static with some nested transient boolean. I expect the whole structure to be always there.
@yenda  I don't know why but my key disappear (not `transactions` as it is already used). Not investigated much as I want to merge this quick.

Reply:
If we don't merge it this week we got to create a whole swarm, oh jeez! :P I'm fine with this.

Reply:
@jeluard I don't understand what you tried, if you put the transaction key in the empty :wallet map in the default app-db (the line above the line this thread refers to), it will be there when you switch account.

Reply:
Not related to switching account, it wasn't there at all for some reason

Reply:
add a todo if you don't want to spend the time on it